backstack/src/commonMain/kotlin/com/slack/circuit/backstack/SaveableBackStack.kt

```diff
@@ -36,7 +36,9 @@ public fun rememberSaveableBackStack(init: SaveableBackStack.() -> Unit): Saveab
  */
 public class SaveableBackStack : BackStack<SaveableBackStack.Record> {
 
-  private val entryList = mutableStateListOf<Record>()
+  // Both visible for testing
+  internal val entryList = mutableStateListOf<Record>()
+  internal val stateStore = mutableMapOf<Screen, List<Record>>()
```

Comment:
Micro-optimization of the century but wanna use [ScatterMap](https://www.romainguy.dev/posts/2024/a-better-hashmap/)?

Reply:
Unfortunately we can't as AndroidX Collection doesn't ship a JS version 😞

Reply:
🪦